utils/api-importer/itn-byron-wallet-importer.js

```diff
@@ -26,7 +26,7 @@ async function main() {
     await Promise.all(mnemonics.map((mnemonic, index) => {
       const name = walletNames[index]
       const payload = generateImportPayload(mnemonic, name)
-      return axios.post(`http://localhost:${API_PORT}/v2/byron-wallets`, payload)
+      return axios.post(`https://localhost:${API_PORT}/v2/byron-wallets`, payload)
```

Comment:
@daniloprates please revert this - we are NOT using https for ITN - there is still no TLS support for Jormungandr.

Reply:
Fixed in: https://github.com/input-output-hk/daedalus/pull/1961/commits/918ba0b5b5a6fb246223d60f22af050fb4e74bf6